server/src/main/java/org/elasticsearch/action/search/SearchRequestBuilder.java

```diff
@@ -549,7 +549,7 @@ public SearchRequestBuilder setCollapse(CollapseBuilder collapse) {
      * @param keepAlive       the extended time to live for the search context
      */
     public SearchRequestBuilder setSearchContext(String searchContextId, TimeValue keepAlive) {
```

Comment:
Should this be renamed to setPointInTime or setPointInTimeBuilder ?

Reply:
++. I pushed https://github.com/elastic/elasticsearch/pull/62720/commits/5af72036b99e9f0c78f116948404f67af036fbe9